File: net.tenteCsharp/src_code/smartEnergyControl/ISmartEnergyNotify.java
package smartEnergyControl;
import smartEnergyControl.SmartEnergyControl.*;

public interface ISmartEnergyNotify{
	
	public void activateSmartControl(String floorId,String roomId);	
	public void deactivateSmartControl(String floorId,String roomId);

}

File: trunk/Elio/pseudoCodeGeneratorElio/src-gen/smartEnergyControl/ISmartEnergyNotify.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface ISmartEnergyNotify
	{
		void activateSmartControl(String floorId, String roomId);
		void deactivateSmartControl(String floorId, String roomId);

	}
}
